core/docker/Dockerfile

```diff
@@ -11,12 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM centos:centos8
+FROM registry.access.redhat.com/ubi8/ubi
 
-ENV JAVA_HOME /usr/lib/jvm/jre-11
+ENV JAVA_HOME /usr/lib/jvm/zulu11
 RUN \
     set -xeu && \
-    yum -y -q install python3 java-11-openjdk-headless java-11-openjdk-devel && \
+    yum -y -q install https://cdn.azul.com/zulu/bin/zulu-repo-1.0.0-1.noarch.rpm && \
+    yum -y -q install python3 zulu11-jdk less && \
```

Comment:
I wonder if we should compare it with the last Trino release to see if it'll be missing some common tools.

It's probably a good replacement for CentOS, and we _have to_ do this now, and it's not the minimal image, but maybe there's something more to be added other than `less`.

Reply:
I can compare. Good point. I intentionally used the non-minimal image to make sure we don't lose much (if at all).

Reply:
Here are more complete lists with `/bin`, `/sbin` etc
[centos.txt](https://github.com/trinodb/trino/files/7972914/centos.txt)
[Uploading ubi.txt…]()

Reply:
<details>
<summary>see diff (click here)</summary>

```diff
diff --git a/tmp/new b/tmp/old
index 71f09860ec..78518d46f0 100644
--- a/tmp/ubi8
+++ b/tmp/centos
@@ -2,30 +2,28 @@ acl
 audit-libs
 basesystem
 bash
-brotli
+bc
+bind-license
+binutils
 bzip2-libs
 ca-certificates
+centos-release
 chkconfig
-coreutils-single
+coreutils
+cpio
 cracklib
 cracklib-dicts
-crypto-policies
-crypto-policies-scripts
 cryptsetup-libs
 curl
 cyrus-sasl-lib
 dbus
-dbus-common
-dbus-daemon
 dbus-glib
 dbus-libs
-dbus-tools
+dbus-python
 device-mapper
 device-mapper-libs
-dmidecode
-dnf
-dnf-data
-dnf-plugin-subscription-manager
+diffutils
+dracut
 elfutils-default-yama-scope
 elfutils-libelf
 elfutils-libs
@@ -34,91 +32,74 @@ file-libs
 filesystem
 findutils
 gawk
-gdb-gdbserver
 gdbm
-gdbm-libs
+geoipupdate
 glib2
 glibc
 glibc-common
-glibc-minimal-langpack
 gmp
 gnupg2
-gnutls
 gobject-introspection
 gpg-pubkey
 gpg-pubkey
-gpg-pubkey
 gpgme
 grep
+groff-base
 gzip
-ima-evm-utils
+hardlink
+hostname
 info
+iputils
 json-c
-json-glib
 keyutils-libs
+kmod
 kmod-libs
+kpartx
 krb5-libs
-langpacks-en
 less
 libacl
-libarchive
 libassuan
 libattr
 libblkid
 libcap
 libcap-ng
 libcom_err
-libcomps
 libcurl
 libdb
 libdb-utils
-libdnf
-libfdisk
 libffi
 libgcc
 libgcrypt
 libgpg-error
-libidn2
-libksba
-libmodulemd
+libidn
 libmount
-libnghttp2
-libnl3
-libnsl2
-libpsl
 libpwquality
-librepo
-libreport-filesystem
-librhsm
-libseccomp
 libselinux
 libsemanage
 libsepol
-libsigsegv
 libsmartcols
-libsolv
-libssh
-libssh-config
+libssh2
 libstdc++
 libtasn1
-libtirpc
-libunistring
-libusbx
 libuser
 libutempter
 libuuid
 libverto
-libxcrypt
 libxml2
-libyaml
-libzstd
-lua-libs
-lz4-libs
-mpfr
+libxml2-python
+lua
+lz4
+ncurses
 ncurses-base
 ncurses-libs
-nettle
-npth
+nspr
+nss
+nss-pem
+nss-softokn
+nss-softokn-freebl
+nss-sysinit
+nss-tools
+nss-util
 openldap
 openssl-libs
 p11-kit
@@ -126,69 +107,49 @@ p11-kit-trust
 pam
 passwd
 pcre
-pcre2
-platform-python
-platform-python-pip
-platform-python-setuptools
+pinentry
+pkgconfig
 popt
-publicsuffix-list-dafsa
-python3-chardet
-python3-cloud-what
-python3-dateutil
-python3-dbus
-python3-decorator
-python3-dmidecode
-python3-dnf
-python3-dnf-plugins-core
-python3-ethtool
-python3-gobject-base
-python3-gpg
-python3-hawkey
-python3-idna
-python3-iniparse
-python3-inotify
-python3-libcomps
-python3-libdnf
-python3-librepo
-python3-libs
-python3-libxml2
-python3-pip
-python3-pip-wheel
-python3-pysocks
-python3-requests
-python3-rpm
-python3-setuptools
-python3-setuptools-wheel
-python3-six
-python3-subscription-manager-rhsm
-python3-syspurpose
-python3-urllib3
-python36
+procps-ng
+pth
+pygpgme
+pyliblzma
+python
+python-chardet
+python-gobject-base
+python-iniparse
+python-kitchen
+python-libs
+python-pycurl
+python-urlgrabber
+pyxattr
+qrencode-libs
 readline
-redhat-release
 rootfiles
 rpm
 rpm-build-libs
 rpm-libs
+rpm-python
 sed
 setup
 shadow-utils
-sqlite-libs
-subscription-manager
-subscription-manager-rhsm-certificates
+shared-mime-info
+sqlite
 systemd
 systemd-libs
-systemd-pam
 tar
-tpm2-tss
 tzdata
-usermode
+ustr
 util-linux
 vim-minimal
-virt-what
-which
+xz
 xz-libs
 yum
+yum-metadata-parser
+yum-plugin-fastestmirror
+yum-plugin-ovl
+yum-plugin-versionlock
+yum-utils
 zlib
 zulu-repo
 zulu11-ca-doc
```

</details>

Reply:
bc, hostname, procps-ng (ps etc.), xz are the only notable ones. I think it's fine as is.